### DISCUSSION TITLE
Custom field for URL

### DISCUSSION THREAD

**Discussion by iampremo** [community/community#69905] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Right now if you want to have a url displayed in the side bar you would need to add it into a text field. This is fine if you only want to look at the url but if you want to actually click on it then you are going to be in for a hard time. Clicking on the link will put you into edit mode for the text field and make clicking the link impossible.

It would be highly beneficial to be able to click on any link in a text field or to have a custom field type of URL that will only accept a url and would make clicking on it possible without entering edit mode.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Julian-Dumitrascu** [comment]
I can follow the links from a [pick list](https://docs.github.com/en/issues/planning-and-tracking-with-projects/understanding-fields/about-single-select-fields).

**Reply by iampremo** [reply]
That kind of works as work around but requires a lot of management, if each issue were to need a different link then it will get added to the list which you then have to search through to add.
